api/admin/controller/collection_settings.py

```diff
@@ -24,10 +25,11 @@ def _get_collection_protocols(self):
         protocols = super(CollectionSettingsController, self)._get_collection_protocols(self.PROVIDER_APIS)
         # If there are storage integrations, add a mirror integration
         # setting to every protocol's 'settings' block.
-        mirror_integration_setting = self._mirror_integration_setting()
-        if mirror_integration_setting:
+        mirror_type = ['Covers', 'Books']
```

Comment:
These should be class constants so that it's easy to see where the configuration UI comes from. This is even more important if you end up sending the label and description into `_mirror_integration_setting`.